scripts/approve_registrations.py

```diff
@@ -51,17 +53,21 @@ def main(dry_run=True):
             if pending_registration.archiving:
                 continue
 
-            with transaction.atomic():
-                try:
-                    # Call 'accept' trigger directly. This will terminate the embargo
-                    # if the registration is unmoderated or push it into the moderation
-                    # queue if it is part of a moderated registry.
-                    registration_approval.accept()
-                except Exception as err:
-                    logger.error(
-                        'Unexpected error raised when approving registration for '
-                        'registration {}. Continuing...'.format(pending_registration))
-                    logger.exception(err)
+            sid = transaction.savepoint()
+            try:
+                # Call 'accept' trigger directly. This will terminate the embargo
+                # if the registration is unmoderated or push it into the moderation
+                # queue if it is part of a moderated registry.
+                registration_approval.accept()
+                transaction.savepoint_commit(sid)
+            except Exception as err:
+                logger.error(
+                    f'Unexpected error raised when approving registration for '
+                    f'registration {pending_registration._id}. Continuing...'
+                )
+                sentry.log_message(str(err))
```

Comment:
Sentry already combines similar errors, it's better to report all the errors we throw. We have plenty of errors on Sentry now that have been reported N number of times where N is over a million. Can you show me anywhere in the code base we gather sentry errors like that? Regardless seems difficult for we to test error aggregation locally and it took very long to set-up Sentry for pigeon, so it doesn't seem worth setting up an instance of Sentry to test locally, kind of stuck with what we have until we can test it with the actual service.

Reply:
@mfraezz any thoughts on this? My understanding is that logging individually would make it easier to get specifics on each failure case and Sentry should auto-aggregate. Is there a particular reason you were wanting one error per script execution? And what would you like to go in the error?

Reply:
Seems like something got lost in translation, though it was originally my misreading that caused the confusion when @jwalz and I discussed -- Sentry will collate based on message content, so if the message content contained guids it could give a distinct error per guid, but I conflated the above `logger.error` call with the `sentry.log_message` call.

If we want the full traceback, however, I think this should be a `sentry.log_exception` instead of `_message`